### ISSUE TITLE
Depth estimation 

### ISSUE THREAD

**Issue by abhishekbu** [priya-dwivedi/Deep-Learning#107]
Hi mam, in depth estimation how to calculate the final depth in metres for every pixel?

**Comment by priya-dwivedi** [comment]
typically this is provided in the training set. There is an estimation of
how depth colors correlate with actual distances.

**Comment by abhishekbu** [comment]
I am using nyu depth dataset v2,
The values in the output of the network are normalized how to unnormalize them? With what value do i need to multiply.

**Comment by priya-dwivedi** [comment]
@abhishekbu - Don;t remember of the top of my head. But check the functions in utils.py

https://github.com/priya-dwivedi/Deep-Learning/blob/master/depth_estimation/utils.py

pred_y = scale_up(2, predict(model, x/255, minDepth=10, maxDepth=1000, batch_size=bs)[:,:,:,0]) * 10.0
